src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+class Circle extends Figure {
```

Comment:
The class Circle should be declared as public to be accessible from outside of its package.